src/main/java/uk/gov/pay/products/resources/ProductMetadataResource.java

```diff
@@ -69,4 +69,14 @@ public Response updateMetadata(@PathParam("productExternalId") String productExt
                            return Response.status(Response.Status.OK).build();
                        }));
     }
+
+    @DELETE
+    @Path("/{metadataKey}")
+    public Response deleteMetadata(@PathParam("productExternalId") String productExternalId, @PathParam("metadataKey") String metadataKey) {
+        logger.info("Delete product metadata for id - [ {} ] with key id [ {} ]", productExternalId, metadataKey);
+
+        return metadataFactory.metadataDeleter().deleteMetadata(productExternalId, metadataKey) ?
+                Response.status(Response.Status.OK).build() :
+                Response.status(Response.Status.NOT_FOUND).build();
```

Comment:
Does this `NOT_FOUND` builder actually get called or does this pass because the method throws `MetadataNotFoundException` ? 

If the latter I think this could be removed as I think there's no case where it's not found and has returned false

Reply:
done!